### PR TITLE
Initial Organization README.md

### PR DESCRIPTION
**Pull request by borozcod**
## Why
It has become challenging to explain what OpenMBEE is and the different architectures. Having something on the github Organization page will communicated some quick facts about who we are and what we do. 
## Updates made
- Followed the guidelines established [here](https://github.blog/changelog/2021-09-14-readmes-for-organization-profiles/). 
- Added a logo
- Added Organization profile README.md
- Added repo README.md

I'm not expecting this pull request will get merged right away. I want to open this up for discussing what should be shown on our GitHub Organization homepage. This is my first pass at it.

### CODE REVIEW

**Comment by borozcod** [comment]
For future work, we can use this repository to set up organization pull request templates
- https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file
- https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates

#2

**Comment by dlamoris** [comment]
notes from tagup:
there should be some mention of tmt, comodo or the other model centric repos (maybe they belong under exec? not sure)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# .github
+[OpenMBEE](https://github.com/Open-MBEE) organization health files
```
Comment (lewismc) [inline README.md:2]: I think this can be removed.

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+![Alternative Text](./img/openmbee-logo.svg)
```
Comment (lewismc) [inline profile/README.md]: We will want to resize this. I think we can do something like...

``

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Welcome to OpenMBEE
+OpenMBEE (Open Model Based Engineering Environment) is an open source collaborative engineering system. It enables engineers to work in the language of their choice and easily share and document their work across other tools.
+
+## OpenMBEE Architectures
+### Execubot
```
Comment (lewismc) [inline profile/README.md]: I think we should put **Flexo** first followed by **Execubots**.

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+The Execubot architecture allows Systems Engineers to develop models in their enterprise model development platform and publish aspects of the model to the web.  
+
+Popular Repositories:
+- [Model Management System](https://github.com/Open-MBEE/mms) - Management and version control for SysML models
+- [Cameo Model Development Kit](https://github.com/Open-MBEE/mdk) - Cameo Systems Modeler plugin for defining views for View Editor
+- [View Editor](https://github.com/Open-MBEE/ve) - Interact with SysML models within a web-based environment
+
+### Flexo
```
Comment (lewismc) [inline profile/README.md:10]: Maybe qualify this with
```
### Flexo (current mainstream development)
```

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Flexo expands on the concept of version control models from execubot. Execubots representation of the data is in JSON, Flexo exposes models as RDF. Models in Flexo are not limited to enterprise model development platforms. A model in Flexo can be anything that is represented as RDF.
+
+Popular Repositories:
```
Comment (lewismc) [inline profile/README.md]: I think we can remove this and include these in the _**pinned repositories**_...

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+### Flexo
+Flexo expands on the concept of version control models from execubot. Execubots representation of the data is in JSON, Flexo exposes models as RDF. Models in Flexo are not limited to enterprise model development platforms. A model in Flexo can be anything that is represented as RDF.
+
+Popular Repositories:  
+- [mms5-layer1-service](https://github.com/Open-MBEE/mms5-layer1-service) - REST server allowing for CRUD operations of named graphs
+- [openapi-graph-extractor](https://github.com/Open-MBEE/openapi-graph-extractor) - Converts well-defined JSON data to RDF by consuming an OpenAPI document 
+- [mms5-deployment](https://github.com/Open-MBEE/mms5-deployment) - YAML files for deployment of MMS5 services in k8s
+
+## Contributing 
+- [Geneal Contributing Guidelines](https://www.openmbee.org/contribute.html#)
```
Comment (lewismc) [inline profile/README.md]: Please correct spelling of **Geneal** --> **General**

**Review by lewismc — CHANGES_REQUESTED** [review]
Thanks for considering the changes.

File: README.md (hunk 1 above)
Comment (borozcod) [reply README.md:2]: I think we should keep it, just to have a README.md for the repo itself.

File: profile/README.md (hunk 4 above)
Comment (borozcod) [reply profile/README.md:10]: I think we should have a roadmap before we add this

File: profile/README.md (hunk 5 above)
Comment (borozcod) [reply profile/README.md]: How about doing both?

**Comment by borozcod** [comment]
> notes from tagup: there should be some mention of tmt, comodo or the other model centric repos (maybe they belong under exec? not sure)

@dlamoris I added a section for example models

File: profile/README.md (hunk 5 above)
Comment (dlamoris) [reply profile/README.md]: given that we plan to rename most of these repos, i think we can finalize this at the openmbee meeting, then there's some pattern we can give for identifying which repo belongs to which architecture

**Pull request merged**